manga_auto_translator/pipeline.py

```diff
@@ -18,13 +27,21 @@ def run(self):
         self.postprocess_scan()
 
     def segmentation(self):
-        pass
+        print('segmentation : ')

```

Comment:
If these print statements are for showing an example when doing inference, we should do this in another PR I think. Also, we should probably add an argument to the CLI args to enable/disable this feature.